.travis.yml

```diff
@@ -10,11 +10,12 @@ addons:
       # Needed for NetMHC
       - tcsh
 cache:
-  pip: true
-  # cache directory used for Ensembl downloads of GTF and FASTA files
+  # cache pip files, also directory used for Ensembl downloads of GTF and FASTA files
```

Comment:
I don't know if you can have multiple `cache` entries - maybe you can? I was mostly looking at "Caching arbitrary directories" and an example listed in https://docs.travis-ci.com/user/caching/#before_cache-phase